docs/semgrep-supply-chain/glossary.md

```diff
@@ -53,19 +53,19 @@ A package registry stores dependencies and provides a means to upload or downloa
 
 A reachable finding means that you are using both a vulnerable piece of code (the **usage**) and the vulnerable version of a dependency. Within Semgrep Supply Chain, specific findings (usages) are grouped together by their vulnerability.
 
-Continuous integration scans with Semgrep Supply Chain rules block any merge or pull requests upon detecting any reachable findings.
+Continuous integration scans with Semgrep Supply Chain rules can block pull or merge requests upon detecting any reachable findings.
 
 See also [Reachability](#reachability).
 
 ## Reachability
 
-Reachability refers to whether or not a vulnerable piece of code from a dependency is used in the codebase itself. In Semgrep Supply Chain, both a dependency's vulnerable version and code pattern must match for a vulnerability to be considered reachable.
+Reachability refers to whether or not a vulnerable piece of code from a dependency is used in the related codebase. In Semgrep Supply Chain, both a dependency's vulnerable version and code pattern must match for a vulnerability to be considered reachable.
```

Comment:
```suggestion
Reachability refers to whether or not a vulnerable piece of code from a dependency is used in the codebase importing it. In Semgrep Supply Chain, both a dependency's vulnerable version and code pattern must match for a vulnerability to be considered reachable.
```

Not sure if my suggestion is any better, tbh.

Reply:
I think it's good, but should we go with "importing it" or "that imports it"? Probably want to use the same here and below.

Reply:
I went with "that imports it" and updated the suggestion accordingly. Thank you!